README.md

```diff
@@ -11,13 +11,13 @@
 
 ## Morpho's vaults
 
-Morpho's vaults represent tokenized supply positions on Morpho. Vaults are compliant to the ERC4626 standard and can be easily integrated. Please refer to the [vaults documentation](https://developers.morpho.xyz/vaults) for more information.
+Morpho's vaults represent tokenized supply positions on Morpho. Vaults are compliant to the ERC4626 standard and can be easily integrated. Please refer to the [vaults documentation](https://developers-vaults.morpho.xyz/) for more information.
 
 ---
 
 ## Audits
 
-All audits are stored in the [audits](./audits/)' folder.
+All audits are accessible in the [audits](https://docs.morpho.xyz/security/audits)' section related to security.
```

Comment:
Do we really want to point to all audits or just the ones related to the vaults that are in this repo?

Reply:
We can say both:
```suggestion
Vaults' audits are accessible in the [audits](./audits/)' folder.
All Morpho audits are accessible in the [audits section of the Morpho documentation](https://docs.morpho.xyz/security/audits).
```

Reply:
Unfortunately, we can't as once deployed, when clicking on the link redirecting to the folder,
it redirects to a 403 Forbidden error: AccessDenied. 
And if we redirect directly to the 2 links, this is the same.

<img width="807" alt="image" src="https://user-images.githubusercontent.com/75627352/227968015-97c98ed6-3807-4c03-9640-6bc99d9c09b2.png">